src/createServer.js

```diff
@@ -1,3 +1,66 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const textToTransform = req.url.slice(1).split('?')[0];
+    const allowedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+    const queryString = req.url.slice(1).split('?')[1];
+    const params = new URLSearchParams(queryString);
+    const toCase = params.get('toCase');
```

Comment:
Move it to the separate function
```suggestion
    const ... = ...( ... );
    const allowedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];

```